### DISCUSSION TITLE
GNSS S-JTSK (EPSG 5514) projection problem

### DISCUSSION THREAD

**Discussion by cz-ranger** [opengisch/QField#2223] [Q&A]
Hello,
I use Qfield on Android with South S660 GNSS receiver which outputs WGS84 data. All my projects use EPSG 5514 projection.  Lately ( a few months ) the positioning is about 1 meter off in bot coordinates. The project file has not changed, nor I did any changes to GPS settings, I just updated Qfield. I also tried connecting GPS directly to Qfield which resulted in location showing completely elswehere (another space dimension, probably). Is there anything else other than converting all projects to use EPSG4326 ? What exactly has changed ? Thanks, Tom

**Comment by nirvn** [comment]
@cz-ranger , can you attach a screenshot of your positioning settings panel? 

Ie:
![Screenshot_20211012-220425_QField Beta 2219](https://user-images.githubusercontent.com/1728657/136981376-070575c7-a24a-4026-841e-6d5aedcbb9e9.jpg)

**Comment by cz-ranger** [comment]
This is the positioning setting using directly connected to GPS via Bluetooth (location totally off)
![Screenshot_20211013-104804](https://user-images.githubusercontent.com/56651714/137101734-8c03f320-5364-4ec5-a4a8-1de7a2c6efd0.jpg)
![Screenshot_20211013-105229](https://user-images.githubusercontent.com/56651714/137101991-7bbbba4a-e99b-40ee-b428-d0fa010822a1.jpg)
This is the position setting when using mock location via native app from South (LBS Manage). This works, but the position is consistently about 1m off in both x and y.  This problem is exclusive to Qfield, in other mapping apps the location appears to be correct.
![Screenshot_20211013-104623](https://user-images.githubusercontent.com/56651714/137101739-31dcb554-64d5-4f9f-a89b-44bf6d246417.jpg)
![Screenshot_20211013-105405](https://user-images.githubusercontent.com/56651714/137102338-46df6db1-7c57-4e3c-91c3-744bce2ac7c2.jpg)

**Comment by nirvn** [comment]
Could you check whether the location appears to be correct if you set your project CRS to EPSG:4326 (ie WGS84)?

**Comment by nirvn** [comment]
@cz-ranger , I did a couple of tests locally. It could well be that your project's datum transformations settings aren't properly set.

If you open your project on QGIS and open the project properties' transformation panel, you can add/edit the transformation maths used to go to/from EPSG:5514 to EPSG:4236 (WGS84, the position's coordinates projection).

This transformation worked for me:
![image](https://user-images.githubusercontent.com/1728657/137125469-17484324-91e8-42d5-b39b-3a00da214230.png)

While the other ones would have test points not align as expected.

I suggest you give this a try. Also, give QField 2.0 beta 8 a try if you haven't yet.
